Documentation/storage-providers.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Storage Providers
+weight: 12050
+indent: true
+---
+
+# Storage Providers
+
+Rook is the home for operators for multiple storage providers. Each of these storage providers
+has specific requirements and each of them is very independent. There is no runtime dependency
+between the storage providers. Development time is where the storage providers benefit one another.
+
+Rook provides a development framework with a goal of enabling storage providers to create
+operators for Kubernetes to manage their storage layer. As the storage provider community
+grows, we expect this framework to grow as common storage constructs are identified
+that will benefit the community. Rook does not aim to replace other frameworks or
+communities, but to fill gaps not filled by other core projects. Here are
+some examples of other frameworks areas that already meet many community needs:
+
+* [Operator SDK](https://sdk.operatorframework.io/): For creating general operators.
+* [Storage SIG](https://github.com/kubernetes/community/tree/master/sig-storage):
+  For K8s storage API included in K8s, CSI drivers, snapshotting, local storage, and much more.
+* [Controller Runtime](https://github.com/kubernetes-sigs/controller-runtime):
+  Set of libraries for building Controllers for K8s.
+
+Rook does not aim to be a general framework for storage, but to provide a
+very specific set of helper to meet the storage provider needs in the Rook project.
```

Comment:
"helpers" rather than "helper".
It's also not clear from this what kind of helpers are provided. But as this is more of an introduction, it may be better to remain vague here and fill in details later.

Reply:
Yes, it's definitely high level, but I'll fix the typo.